rosettanet/2.0/modules/ROOT/pages/rosettanet-connector-examples.adoc

```diff
@@ -445,11 +446,11 @@ To test the Buyer and Seller apps, do the following after you run the apps:
 
 Use an HTTP POST to the Buyer app `HTTP_InputListener` endpoint to provide the purchase order request to send to the seller.
 
-You can download a https://s3-us-west-2.amazonaws.com/mulesoft-sites-vendorcontent/public-assets/sample-purchase-order-request-content.xml[sample purchase order request]. You can then use any HTTP tool, such as a browser plugin, standalone tool such as PostMan, console tool such as `curl` to POST the data to the Buyer app.
+You can download a https://s3-us-west-2.amazonaws.com/mulesoft-sites-vendorcontent/public-assets/sample-purchase-order-request-content.xml[sample purchase order request]. You can then use any HTTP tool, such as a browser plugin, standalone tool such as PostMan, or console tool such as `curl` to POST the data to the Buyer app.
 
 For example, the following `curl` command posts a purchase order request:
 
-`+curl -v -H "Content-Type: application/text" -XPOST --data-binary @sample-purchase-order-request-content.xml http://localhost:8801+`
+`+curl -v -H "Content-Type: application/text" -XPOST --data-binary @sample-purchase-order-request-content.xml http://localhost:8081+`
```

Comment:
This should be port `8801` as this curl is to trigger the send operation flow, I would also change the `application/text` to `application/xml` 
```suggestion
`+curl -v -H "Content-Type: application/xml" -XPOST --data-binary @sample-purchase-order-request-content.xml http://localhost:8801+`
```